risc0/zkvm/src/host/server/exec/tests.rs

```diff
@@ -839,6 +1150,6 @@ mod docker {
         let err = run_session(1 << 16, 15, 3).err().unwrap();
         assert!(err.to_string().contains("Session limit exceeded"));
 
-        assert!(run_session(1 << 16, 15, 10).is_ok());
+        assert!(run_session(1 << 16, 15, 16).is_ok());
```

Comment:
@flaub it appears that the extra work in the guest, presumably the slight increase in data hashed for the output plus any paging, increased the minimum number of cycles are reduced the amount of work that can be done in a 2^15 segment by a significant amount. I want to investigate this further, but would also like to get this PR merged.

Should we go ahead and merge this now?